spec/features/admin/allocation_chart_spec.rb

```diff
@@ -3,6 +3,7 @@
 require 'rails_helper'
 
 describe 'Admin Allocation chart', type: :feature do
+  next_month_date = Date.today.next_month
```

Comment:
is not a good practice to use local variables outside examples and before, please use let instead